### PR TITLE
init of the project

### PR DESCRIPTION
**Pull request by datanel**
For the moment it only watch a directory and move all the files to another directory.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+amqp==1.4.9
+anyjson==0.3.3
+billiard==3.3.0.23
```
Comment (kinnou02) [inline requirements.txt]: You should use the current version of all your dependencies

File: tartare/default_settings.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import logging
+from datetime import timedelta
+
+#URL for the brokker, by default it's the local rabbitmq
+#For amqp (rabbitMQ) the syntax is:
+#amqp://<user>:<password>@<host>:<port>/<vhost>
+#the default vhost is "/" so the URL end with *two* slash
+#http://docs.celeryproject.org/en/latest/configuration.html#std:setting-BROKER_URL
+CELERY_BROKER_URL = 'amqp://guest:guest@localhost:5672//'
+
```
Comment (kinnou02) [inline tartare/default_settings.py]: you will have to specify the queue uses by celery, by default it's "CELERY" we don't want to conflict with tyr

File: tartare/default_settings.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+}
+CELERY_TIMEZONE = 'UTC'
+
+#http://docs.celeryproject.org/en/master/configuration.html#std:setting-CELERYBEAT_SCHEDULE_FILENAME
+CELERYBEAT_SCHEDULE_FILENAME = '/tmp/celerybeat-schedule'
+
+CELERYD_HIJACK_ROOT_LOGGER = False
+
+INPUT_DIR = './input'
+OUTPUT_DIR='./output'
```
Comment (antoine-de) [inline tartare/default_settings.py]: you need a return line here

File: tartare/default_settings.py (hunk 2 above)
Comment (datanel) [inline tartare/default_settings.py]: done

File: requirements.txt (hunk 1 above)
Comment (datanel) [inline requirements.txt]: done

File: tartare/default_settings.py (hunk 3 above)
Comment (datanel) [inline tartare/default_settings.py]: done

**Comment by antoine-de** [comment]
Ok for me

**Pull request merged**